### PR TITLE
fix backups

### PR DESCRIPTION
**Pull request by LegendreM**
* pluralize variable `backup_folder` -> `backups_folder`
* change env case `MEILI_backup_folder` -> `MEILI_BACKUPS_FOLDER`
* add miliseconds to backup ID to reduce colisions

### CODE REVIEW

**Review by curquiza — CHANGES_REQUESTED** [review]
You changed the `movies.json` file when changing backup into dump 😉

File: meilisearch-http/src/dump.rs [hunk 1]
```diff
@@ -143,28 +143,35 @@ fn import_index_v1(
...
-/// Import backup from `backup_folder` in database.
-pub fn import_backup(
+/// Import dump from `dump_path` in database.
+pub fn import_dump(
     data: &Data,
-    backup_folder: &Path,
+    dump_path: &Path,
     document_batch_size: usize,
 ) -> Result<(), Error> {
+    eprintln!("Import dump from {:?}", dump_path);
```
Comment (MarinPostma) [inline meilisearch-http/src/dump.rs:160]: can you use the macros from the log crate instead? (log, warn etc...)

File: meilisearch-http/src/dump.rs [hunk 2]
```diff
@@ -184,78 +191,79 @@ pub fn import_backup(
         Ok(())
     })?;
 
+    eprintln!("Dump importation from {:?} succeed", dump_path);
```
Comment (MarinPostma) [inline meilisearch-http/src/dump.rs:194]: same here

File: meilisearch-http/src/dump.rs (hunk 1 above)
Comment (MarinPostma) [inline meilisearch-http/src/dump.rs:160]:
```suggestion
    eprintln!("Importing dump from {:?}...", dump_path);
```

File: meilisearch-http/tests/dump.rs [hunk 3]
```diff
@@ -126,25 +126,25 @@ async fn get_backup_status_early_should_return_processing() {
...
+async fn get_dump_status_should_return_done() {
     let mut server = common::Server::test_server().await;
 
 
-    let (value, status_code) = server.trigger_backup().await;
+    let (value, status_code) = server.trigger_dump().await;
 
     assert_eq!(status_code, 202);
 
     println!("{:?}", value);
```
Comment (MarinPostma) [inline meilisearch-http/tests/dump.rs:137]: oops debug message

**Review by MarinPostma — CHANGES_REQUESTED** [review]

**Review by MarinPostma — APPROVED** [review]
can you squash everything in one commit please?

**Comment by MarinPostma** [comment]
Bors merge

**Pull request merged**